package.json

```diff
@@ -11,7 +11,7 @@
     "unlink": "cd packages/aeproject-cli && npm unlink && cd .. && cd aeproject-lib && npm unlink && npm unlink aeproject-config && npm unlink aeproject-utils && npm unlink aeproject-logger && cd .. && cd aeproject-config && npm unlink && cd .. && cd aeproject-utils && npm unlink && npm unlink aeproject-logger && cd .. && cd aeproject-logger && npm unlink"
   },
   "dependencies": {
-    "@aeternity/aepp-sdk": "7.7.0",
+    "@aeternity/aepp-sdk": "8.0.0-beta.2",
```

Comment:
```suggestion
    "@aeternity/aepp-sdk": "^8.0.0",
```

Reply:
yeah I will soon commit a new state including the updated SDK